src/containers/account/TokenTransferForm.js

```diff
@@ -149,7 +150,13 @@ class TokenTransferForm extends React.Component {
                             placeholder="short narrative (optional)"
                             disabled={submitting || !augmintToken.isConnected}
                         />
-                        <Button loading={submitting} primary disabled={pristine} data-testid="submitTransferButton">
+                        <Button
+                            type="submit"
+                            loading={submitting}
+                            primary
+                            disabled={pristine}
```

Comment:
is `disabled` available in the `augmint-ui/button` component?

Reply:
I forgot it...
we have many button component implementations in the different branches... 😃
I fix it